metrics/api/v1/model_handlers.go

```diff
@@ -33,14 +33,6 @@ import (
 // when manager.model has not been initialized.
 var errModelNotActivated = errors.New("the model is not activated")
 
-var metricNamesConversion = map[string]string{
```

Comment:
Unfortunately you cannot remove cpu-usage mapping. This is a legacy constant used by Horizontal Pod Autoscaler when talking to Heapster.